data/readreplica.py

```diff
@@ -71,6 +70,32 @@ def execute_sql(self, sql, params=None, commit=SENTINEL):
                     raise
 
 
+class DoubleWriteWrapper(AutomaticFailoverWrapper):
+
+    def __init__(self, primary_db, fallback_db, secondary_write_db=None):
+        super().__init__(primary_db, fallback_db)
+        self._secondary_write_db = secondary_write_db
+
+    def __getattr__(self, attribute):
+        return super().__getattr__(attribute)
+
+    def execute(self, query, commit=SENTINEL, **context_options):
+        ctx = self.get_sql_context(**context_options)
+        sql, params = ctx.sql(query).query()
+        return self.execute_sql(sql, params, commit=commit)
+
+    def execute_sql(self, sql, params=None, commit=SENTINEL):
+        result = super().execute_sql(sql, params, commit)
+        # TODO: remove try when we tie primary and secondary together under transaction
+        #try:
+        self._secondary_write_db.execute_sql(sql, params, commit)
```

Comment:
null check for `_secondary_write_db`

Reply:
this is in the cleanup it was removed during debugging.